### ISSUE TITLE
Add item count to applicable tab labels

### ISSUE THREAD

**Issue by brashrebel** [easydigitaldownloads/easy-digital-downloads#6358]
Our card interface with vertically stacked left side tabs used in a few places (such as the individual customer view) could be enhanced just a little by including an item count on the labels of views which are list based.

Example:
<img width="292" alt="screen shot 2018-02-06 at 11 08 32 am" src="https://user-images.githubusercontent.com/4993131/35869843-21e77248-0b2e-11e8-8754-7fb53c262389.png">
Or perhaps a more subtle look:
<img width="294" alt="screen shot 2018-02-06 at 11 12 01 am" src="https://user-images.githubusercontent.com/4993131/35870031-9c15ca56-0b2e-11e8-9e11-9b88d8182e01.png">

The idea is that some of the views listed as tabs represent tools while others represent lists. The experience on this interface could be improved by showing an item count at all times. As it is now, there's no way to know whether a customer record has notes unless you visit that view.

Other examples of views which could apply:

- License - > Logs
- Vendors - > Notes
- Vendors - > Products
- Vendors - > Commissions

Can't think of other instances of this card/tab view though wouldn't be surprised if things like subscriptions, all access passes and more adopt it in the future.

Existing inspiration to look at would be WP core's available updates count in the toolbar and admin menu as well as Fraud Monitor's payment's in moderation count next to the submenu label.

**Comment by cklosowski** [comment]
I like this idea. As far as implementation, it should all be filter run. So that each tab had a 'count' filter. If that count is greater than `0` or non-empty, we can drop a count.

**Comment by SDavisMedia** [comment]
@cklosowski kinda jumping out on a limb with this one. Super curious if it's heading in the right direction, though. Not pressing, but let me know what you think when you get a chance.

**Comment by cklosowski** [comment]
@SDavisMedia go ahead and kick up a PR, will make it easier to review. Just do it against `master` at this point.

**Comment by cklosowski** [comment]
@SDavisMedia I think this is the right approach. Let'd hold off until 3.0 is out though to take this further. There is real possibility that this will be a huge performance drain on loading a customer, as it will have to query the counts on every page view...unless we refactor the card content to be loaded via ajax.

Reloading the sidebar menu tabs will be brutal to run those recounts every time someone changes tabs.

**Comment by brashrebel** [comment]
On a customer call right now and they described a desire for this almost exactly.

**Comment by cklosowski** [comment]
I think my concerns here about having to query this data on every page load will be easier with EDD 3.0. If not perfect, at least better.

**Comment by ashleyfae** [comment]
I like this idea.
